src/models/user.ts

```diff
@@ -319,4 +319,23 @@ export const Users: UserCollection = {
 
     return null;
   },
+
+  async addActiveDayEvent (client, user, origin) {
+    const [exists] = await client('user_account_active_day')
+      .where({
+        user_account_id: user.id,
+        origin,
+      })
+      .andWhereRaw('created_at >= CURRENT_DATE - INTERVAL \'1 day\'');
```

Comment:
This means that the following could happen:
1. User request at 1pm day 0 => active event stored
2. User request at 3am day 1 => no active event stored

Is that what we want? The requests happened on two different days, shouldn't they both be stored?